### ISSUE TITLE
[Minor BUG] Rotator Type Confusion

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#3470]
If a rotator and a hinge are both connected from a single sequencer gate, the angular position is not universally interpreted.

For example, setting the on state angle to 90° will put the rotator at 90°, but the hinge will go to 45°. It looks like the hinge is reducing the target angle pro-rata to suit its reduces maximum range.

**Comment by tsunamayo** [comment]
Yes it is a not really made for that. I have some refactoring planned. I will first get rid of the sequencer and merge it into the rotator setting itself, then I will introduce two new gate to do much more powerful things. thx
